src/Components/App/App.js

```diff
@@ -25,13 +49,13 @@ function App() {
         -
       </button> */}
       <Routes>
-        <Route path="/" element={<Welcome />} />
+        <Route path="/" element={<Login />} />
+        <Route path="/home" element={<Welcome />} />
         <Route path="/trails" element={<AllTrails />} />
+        <Route path="/saved_trails" element={<SavedTrails />} />
+        <Route path="/individual_trail" element={<IndividualTrail />} />
```

Comment:
Definitely not something you need to do for this project, but thought I should mention: It could be worth thinking about how you might condense some routes.

For example, Saved Trails and Individual Trail could both be parts of the `/trails` route. Individual Trail could just be a search parameter on the url, something like `/trails?id=938yfaoifj` and the `<AllTrails />` component could check that search param and render the `<IndividualTrail />` component if it sees it, or all trails if it doesn't.

Just something to think about for future projects.